### ISSUE TITLE
Godot 4 version

### ISSUE THREAD

**Issue by SpyrexDE** [SnoringCatGames/surfacer#5]
Will the addon be ported to Godot 4 and when could this be?

**Comment by levilindsey** [comment]
Yes! But not soon. Sometime 2023. I started a new job a couple months ago, and will need to onramp for a few more before I'll have free time to work on this again.

**Comment by H0W2D017** [comment]
Guess the godot 4 uppgrade sadly never happened.
